app-api-common/recipe/src/test/java/org/codeforvictory/android/recipes/data/api/recipe/AdaptToMealTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.codeforvictory.android.recipes.data.api.recipe
+
+import org.amshove.kluent.shouldBeEqualTo
+import org.codeforvictory.android.recipes.data.api.recipe.model.ApiMeal
+import org.junit.jupiter.api.Test
+
+internal class AdaptToMealTest {
+
+    private val adaptToMeal = AdaptToMeal()
+
+    @Test
+    fun `invoke - should convert api into domain meal data`() {
```

Comment:
man I just can't get used to these backticked function names with spaces :)

Reply:
I think it's easier to read natural language than having to deal with test cases like:

```
invoke_shouldDoThis_whenGivenXYZ
```

Reply:
Cesar has my vote lol

Reply:
Team backticks! 🥇